validation/chOpenApiValidator.go

```diff
@@ -146,42 +140,116 @@ func formatError(contextId string, err error) []byte {
 	return mr
 }
 
-func extractRequestError(re *openapi3filter.RequestError, errsArray []models.CHError) []models.CHError {
+func handleMultiError(contextId string, mea *openapi3.MultiError, errsArray []models.CHError) []models.CHError {
 
-	for _, me := range re.Err.(openapi3.MultiError) {
+	// err is a *MultiError, and mea is set to the error's value, so range over it to grab each error.
+	for _, e := range *mea {
+		// Begin comparing the retrieved error (e) to possible returned error types.
 
-		// Cast to SchemaError so that we can pull out all of the necessary data to build our CH Errors response.
-		schemaError := me.(*openapi3.SchemaError)
-		reason := strings.Replace(schemaError.Reason, "\"", "'", -1)
-		jsonPath := strings.Join(schemaError.JSONPointer(), ".")
-		fieldName := schemaError.JSONPointer()[len(schemaError.JSONPointer())-1]
+		// If we have a request error, pass re into the extractRequestError func to get a formatted CHError back.
```

Comment:
Should this read `into the handleRequestError`?

Reply:
Fixed this, good spot. Also found another comment with an outdated function name so fixed that too in latest commit.